queries/gaptst/highlights.scm

```diff
@@ -0,0 +1,19 @@
+(output_line) @string.documentation
```

Comment:
I am wondering if there is a better capture for this? Not sure if plain `@string` would make it too confusing with regular code but it also doesn't really seem like documentation. But I guess test output is in a way documenting what should happen :thinking:

Reply:
Good question; I don't think we have any guidance for "session/notebook" languages yet.

Unless we come up with something better, I think `@markup.raw.block` would be my suggestion.

Reply:
`@markup.raw.block` definitely conveys the semantics of the `(output_line)` nodes better. They are essentially the raw expected output of the test cases. I initially put `string.documentation` since I couldn't find a better option. Thanks for the suggestion!